src/lib/viem-client.ts

```diff
@@ -0,0 +1,8 @@
+import { mainnet } from '@wagmi/chains'
```

Comment:
Yeah, this is supposed to remove wagmi, but tests are borked if we import `viewm/chains` - which just proxies `@wagmi/chains` as a transient dep